neo/test/controllers/ApplicationTest.scala

```diff
@@ -0,0 +1,18 @@
+package controllers
+
+import java.io.File
+
+import org.specs2.mutable.Specification
+
+/**
+  * Created by zongh on 10/26/2016.
+  */
+class ApplicationTest extends Specification {
+
+  "application" should {
+    "load the city data from a file" in {
+      val cities = Application.loadCity(new File("C:\\Users\\zongh\\OneDrive\\Documents\\UCI\\cloudberry\\cloudberry\\neo\\public\\data\\city.json"))
```

Comment:
@xizzzz can you give @zonghengma a code example (a github link of the code?) about how to read the file inside project by using relative path than absolutely one?